packages/manager/apps/telecom/src/app/telecom/pack/xdsl/meetings/xdsl-meetings.controller.js

```diff
@@ -0,0 +1,176 @@
+import get from 'lodash/get';
+import 'moment';
+
+export default class XdslMeetingCtrl {
+  /* @ngInject */
+  constructor($scope, $translate, OvhApiXdsl, TucToast) {
+    this.$scope = $scope;
+    this.$translate = $translate;
+    this.OvhApiXdsl = OvhApiXdsl;
+    this.TucToast = TucToast;
+  }
+
+  $onInit() {
+    this.select = {
+      meetingSlots: {
+        fakeMeeting: false,
+        slot: null,
+      },
+    };
+
+    this.showMeetingSlots = false;
+    this.meetingSlots = {};
+
+    this.loading = true;
+    this.meetings = [];
+
+    this.errorMessage = '';
+    this.successMessage = '';
+
+    // Retrieve list of meetings available
+    return this.OvhApiXdsl.v6()
+      .searchOrderMeetings(this.$scope, {
+        serviceName: this.serviceName,
+      })
+      .then((data) => {
+        if (data.result) {
+          this.meetingSlots.canBookFakeMeeting = data.result.canBookFakeMeeting;
+          this.meetingSlots.slots = data.result.meetingSlots;
+
+          let slots = [];
+          let prevTitle;
+          data.result.meetingSlots.forEach((slot, index) => {
+            const title = moment(slot.startDate)
+              .utc()
+              .format('ddd DD MMM YYYY');
+            if (!prevTitle) {
+              prevTitle = title;
+              slots.push({
```

Comment:
this `push` item is done in the three conditions, this can be extract from the if...else